README.md

```diff
@@ -280,6 +280,8 @@ scaling:
   redis:
     host:
     password:
+    existingSecret:
```

Comment:
Can you uncomment those two lines, as they are optional.
Also you can add a comment above explaining the purpose of those two entries.